test/src/server/routes/entity/entity.js

```diff
@@ -80,3 +84,391 @@ describe('getDefaultAliasIndex', () => {
+
+		// Get list of Affected Entities
+		const affectedEntities = await deleteRelationships(orm, null, workA.toJSON());
+
+		const affectedEntityRelationshipSet = await new RelationshipSet({
+			id: affectedEntities[0].get('relationshipSetId')
+		}).fetch({withRelated: 'relationships'});
+
+		expect(affectedEntities[0].get('bbid')).to.equal(author.get('bbid'));
+		expect(affectedEntityRelationshipSet.toJSON().relationships[0].targetBbid).to.be.equal(workB.get('bbid'));
+		expect(affectedEntities).to.have.length(1);
+	});
+	it('it should return an array with multiple other entities to modify if there are relationships to multiple entities', async () => {
+		// Create Entites
+		const workABBID = getRandomUUID();
+		const workBBBID = getRandomUUID();
+		const workCBBID = getRandomUUID();
+		const workA = await createWork(workABBID);
+		const workB = await createWork(workBBBID);
+		const workC = await createWork(workCBBID);
+
+		// Create Revision
+		const revisionId = random.number();
+		await new Revision({authorId: editorJSON.id, id: revisionId})
+			.save(null, {method: 'insert'});
+
+		// Create Relations between entity
+		const relationshipDataA = {
+			id: 1,
+			sourceBbid: author.toJSON().bbid,
+			targetBbid: workA.toJSON().bbid,
+			typeId: 1
+		};
+		const relationshipDataB = {
+			id: 1,
+			sourceBbid: author.toJSON().bbid,
+			targetBbid: workB.toJSON().bbid,
+			typeId: 1
+		};
+		const relationshipDataC = {
+			id: 1,
+			sourceBbid: author.toJSON().bbid,
+			targetBbid: workC.toJSON().bbid,
+			typeId: 1
+		};
+
+		// Relate Relationship Data with Relationship Type
+		relationshipDataA.typeId = relationshipTypeData.id;
+		relationshipDataA.id = random.number();
+
+		relationshipDataB.typeId = relationshipTypeData.id;
+		relationshipDataB.id = random.number();
+
+		relationshipDataC.typeId = relationshipTypeData.id;
+		relationshipDataC.id = random.number();
+
+		// Create Relationships for Entities
+		const relationshipA = await new Relationship(relationshipDataA)
+			.save(null, {method: 'insert'});
+
+		const relationshipB = await new Relationship(relationshipDataB)
+			.save(null, {method: 'insert'});
+
+		const relationshipC = await new Relationship(relationshipDataC)
+			.save(null, {method: 'insert'});
+
+		// Create Relationship set ids for entities
+		const authorRelationshipSetId = random.number();
+		const workARelationshipSetId = random.number();
+		const workBRelationshipSetId = random.number();
+		const workCRelationshipSetId = random.number();
+
+		// Create Relationship sets
+		let authorRelationshipSet = await new RelationshipSet({id: authorRelationshipSetId})
+			.save(null, {method: 'insert'})
+			.then(
+				(model) =>
+					model.relationships().attach([relationshipA, relationshipB, relationshipC]).then(() => model)
+			);
+		authorRelationshipSet = await authorRelationshipSet.refresh({withRelated: 'relationships'});
```

Comment:
There's a need to refresh any of the newly created relationship sets. The reason is that we don't actually use use —in this case— authorRelationshipSet anywhere after its creation (apart from getting the id, which will work without the refresh, and could also be done by using the authorRelationshipSetId variable).

The reason you had to refresh earlier was for debugging purposes so that you could see which relationships were in the set, which isn't needed anymore.
Removing all the calls to `.refresh` in this test file will shave off a bit of time when running the tests.